src/main.cc

```diff
@@ -299,6 +300,42 @@ StartEndpoints(
   return true;
 }
 
+uint32_t
+PrepareStopEndpoints(uint32_t timeout_secs)
```

Comment:
```suggestion
WaitForPendingConnections(uint32_t timeout_secs)
```

Reply:
Picked the latter, since they are existing connections.